### PR TITLE
a possible implementation of the guides landing page idea

### PR DESCRIPTION
**Pull request by MelSumner**
This is just one possible way we could implement a landing page to the guides. I'm not sure we should ship this, but I want us to start the conversation since it's been on our to-do list since last March. 

image

### CODE REVIEW

File: guides/release/index.md [hunk 1]
```diff
@@ -1,117 +1,28 @@
...
+Welcome to the Ember.js Guides! This documentation will take you from total beginner to Ember expert.
+
+With the plethora of libraries readily available for front-end development, sometimes it can be a little confusing to work with a front-end framework like Ember.js, where _everything_ you need to build an application is already included. To that end, we've segmented out each part of the guides so you can focus on just the part you want to work with. This should also make it faster for you to find what you need!
+
+<ul class="list-cards">
+	<li class="list-item-card">
+		<div class="shape shape--dark"></div>
+    <div class="shape shape--accent"></div>
+    <div class="shape shape--light"></div>
+		<a href="https://guides.emberjs.com/release/getting-started/intro/">Ember.js</a>
```
Comment (acorncom) [inline guides/release/index.md]: Looks good here, will this link be accurate post merge?

Comment (MelSumner) [reply guides/release/index.md]: I added the intro page, so it should be. I'm not sure that's what it should be named though.

File: guides/release/pages.yml [hunk 2]
```diff
@@ -8,6 +8,8 @@
 - title: "Getting Started"
   url: 'getting-started'
   pages:
+    - title: "How To Use The Guides"
+      url: "intro"
```
Comment (mansona) [inline guides/release/pages.yml]: this should probably be renamed to index and what is currently index should be renamed to installing 😂

File: guides/release/index.md (hunk 1 above)
Comment (mansona) [reply guides/release/index.md]: see my other comment 👍

File: guides/release/pages.yml (hunk 2 above)
Comment (MelSumner) [reply guides/release/pages.yml]: the landing page is just guides.emberjs.com (equiv to guides.emberjs.com/index.html). 

how to use the guides/intro page (I didn't know what to name it) is a new page that I added, and moved the content to this location.

**Comment by MelSumner** [comment]
@jenweber what do you think of this?

**Comment by jenweber** [comment]
I think we should ship it!

**Pull request merged**